integrations/optimizely/package.json

```diff
@@ -46,6 +46,7 @@
     "karma-sauce-launcher": "^2.0.2",
     "karma-spec-reporter": "^0.0.32",
     "karma-summary-reporter": "^1.6.0",
+    "lerna": "^3.15.0",
```

Comment:
Seems to be unnecessary, but I'll let the Segment folks weigh in

Reply:
TBH, I believe it's a mistake on my end